### PR TITLE
[Geant4] Update to 10.7.cand00 (with VecGeom)

### PR DESCRIPTION
**Pull request by mrodozov**
please test

### CODE REVIEW

**Comment by mrodozov** [comment]
please test

File: geant4-data-rpm.file [hunk 1]
```diff
@@ -10,9 +10,11 @@
 
 %if "%{?source0:set}" != "set"
 %define source0  http://geant4.cern.ch/support/source/%{G4DataTool}.%{realversion}.tar.gz
+%define source1 http://cmsrep.cern.ch/cmssw/download/geant4/%{G4DataTool}.%{realversion}.tar.gz
```
Comment (smuzaffar) [inline geant4-data-rpm.file]: @mrodozov , no you should not change this. You should add 
```
%define source0 http://cmsrep.cern.ch/cmssw/download/geant4/<TOOLNAMe>.%{realversion}.tar.gz
```
in each of the `geant4-<TOOL>.spec` for which we have to get data from this special location

File: geant4-data-rpm.file [hunk 2]
```diff
@@ -10,9 +10,11 @@
...
 %endif
 
 Source0: %source0
+Source1: %source1
```
Comment (smuzaffar) [inline geant4-data-rpm.file]: this is not needed. Just set source0 in the `geant4-*.spec` file

**Comment by mrodozov** [comment]
please test

**Comment by mrodozov** [comment]
please test

**Comment by mrodozov** [comment]
I archived the files with -zcvf and copied them again,
when I ran 
tar -C test/ -zxvf G4RealSurface.2.2.tar.gz
under /data/cmssw/download/geant4 on cmsrep,
on this same file - it gots unarchived :/

**Comment by mrodozov** [comment]
please test

**Comment by mrodozov** [comment]
please test

**Pull request closed without merging**